crowbar_framework/spec/controllers/api/backups_controller_spec.rb

```diff
@@ -0,0 +1,146 @@
+require "spec_helper"
+
+describe "Backups API controller", type: :request do
+  let(:fixture) { Rails.root.join("spec", "fixtures", "crowbar_backup.tar.gz") }
+  let(:restore_status) { Rails.root.join("spec", "fixtures", "backup_restore_status.json") }
+  let(:created_at) { Time.zone.now.strftime("%Y%m%d-%H%M%S") }
+  let!(:stub_methods) do
+    allow_any_instance_of(Crowbar::Backup::Export).to receive(:export).and_return(true)
+    allow_any_instance_of(Kernel).to receive(:system).and_return(true)
+    allow_any_instance_of(Api::Backup).to receive(:path).and_return(fixture)
+    allow_any_instance_of(Api::Backup).to receive(:delete_archive).and_return(true)
+    allow_any_instance_of(Api::Backup).to receive(:create_archive).and_return(true)
+  end
+  let!(:backup_attrs) do
+    {
+      name: "crowbar_backup",
+      migration_level: 20151222144603,
+      version: "4.0",
+      size: 30
+    }
+  end
+
+  context "Successful Backups API request" do
+    let(:headers) { { ACCEPT: "application/vnd.crowbar.v2.0+json" } }
+
+    it "lists all crowbar backups" do
+      get "/api/crowbar/backups", {}, headers
+      expect(response).to have_http_status(:ok)
+      expect(response.body).to eq("[]")
+    end
+
+    it "creates a backup" do
+      stub_methods
```

Comment:
> why not to use before(:each) block? If you want opt-in behavior, just wrap your examples subset in context block, and use before inside that context.

in this case a `before(:each)` makes more sense, agreed...

> Do you want me to continue nitpicking your specs, or you're fine with how it is now, and I'm only an obstacle for you to close the current task and move on?

depends if you have the time, and you are not an obstacle at all, on the contrary.
The tests will anyways change in the actual implementation phase of the API, right now this is "just" the base. and I'd definitely prefer to have a good base to start with.